services/tenant-ui/frontend/src/assets/variables.scss

```diff
@@ -0,0 +1,7 @@
+// Theme variables
```

Comment:
smart, adding this file. nice to see things moving to a more maintainable direction.